builder/docker/doc.go

```diff
@@ -0,0 +1,7 @@
+/*
+Package docker provides Docker utlities.
+
+Some of the utilities rely upon the presence of the command line client for
+Docker. Others use the Docker API.
+*/
+package docker
```

Comment:
I saw that in one of the core Go packages, and I like it. But I usually forget to do it.